drivers/interrupt_controller/CMakeLists.txt

```diff
@@ -10,3 +10,4 @@ zephyr_sources_ifdef(CONFIG_SOC_FAMILY_STM32        exti_stm32.c)
 zephyr_sources_ifdef(CONFIG_CAVS_ICTL               cavs_ictl.c)
 zephyr_sources_ifdef(CONFIG_DW_ICTL                 dw_ictl.c)
 zephyr_sources_ifdef(CONFIG_RV32M1_INTMUX           rv32m1_intmux.c)
+zephyr_sources_ifdef(CONFIG_GIC                     gic.c)
```

Comment:
just minor comments, how about the name change to gic-400 to match the implementation?

Reply:
done